### PR TITLE
[clamav] upgrade pip before using it

### PR DESCRIPTION
**Pull request by micahsnyder**
Pip from base image no longer works to install mussels dependency.

### CODE REVIEW

**Review by DonggeLiu — APPROVED** [review]

**Comment by DonggeLiu** [comment]
/gcbrun trial_build.py clamav --sanitizer coverage address --fuzzing-engine libfuzzer

**Comment by micahsnyder** [comment]
(rebased)

**Comment by DonggeLiu** [comment]
/gcbrun trial_build.py clamav --sanitizer coverage address --fuzzing-engine libfuzzer

**Comment by micahsnyder** [comment]
@DonggeLiu it seemed like it failed.  But I don't know what it means.  Can you help?

File: infra/helper.py
Comment (DaveLak) [inline infra/helper.py:753]: Did you find that adding this arg was necessary for fixing your project builds?

I'm not a maintainer here, but I have fixed some builds by upgrading pip and I haven't seen needed to touch anything in `infra/`.

I think removing this should probably make the failing infra related CI checks unnecessary and unblock your PR.

Comment (DonggeLiu) [reply infra/helper.py:753]: Yeah I am wondering the same thing.
Could you please clarify? @micahsnyder

**Comment by DonggeLiu** [comment]
> @DonggeLiu it seemed like it failed. But I don't know what it means. Can you help?

Cloud build is not necessary in this case, I reckon CI requested it because you modified `infra/`.
If you could revert your change in `infra/helper.py`, then we don't have to worry about this.
Thanks

File: infra/helper.py
Comment (micahsnyder) [reply infra/helper.py:753]: I did not intend to commit this change.  I found it is required if you're using docker buildx. I keep this change locally so it works, but didn't mean to add it in my PR.  I will remove it.

Comment (micahsnyder) [reply infra/helper.py]: @DaveLak @DonggeLiu Sorry about that.  Fixed now.

**Pull request merged**
